mypy_django_plugin/transformers/models.py

```diff
@@ -345,15 +345,19 @@ def run_with_model_cls(self, model_cls: Type[Model]) -> None:
                     related_manager_info = self.lookup_typeinfo_or_incomplete_defn_error(
                         fullnames.RELATED_MANAGER_CLASS
                     )  # noqa: E501
-                    default_manager = related_model_info.names.get("_default_manager")
-                    if not default_manager:
-                        raise helpers.IncompleteDefnException()
                 except helpers.IncompleteDefnException as exc:
                     if not self.api.final_iteration:
                         raise exc
                     else:
                         continue
 
+                default_manager = related_model_info.names.get("_default_manager")
+                if not default_manager:
+                    self.add_new_node_to_model_class(
+                        attname, Instance(related_manager_info, [Instance(related_model_info, [])])
+                    )
```

Comment:
What if we move this piece inside the `else` block on line 351. Together with emitting an error? I think that feels most correct, as the model's manager can't be resolved, an error is displayed, and the default manager type attached still includes defaults from Django.

Emitting an error there feels quite helpful IMO, for debugging etc.

Reply:
This seems like a fair approach.
Do we have any prior art on emitting errors/warnings for certain nodes?

Reply:
ctx.api.fail it seems.